src/main/java/sirius/biz/importer/txn/ImportTransactionData.java

```diff
@@ -27,7 +27,6 @@
      * Represents the last import transaction id which was used to modify the owning entity.
      */
     public static final Mapping TXN_ID = Mapping.named("txnId");
-    @AutoImport
```

Comment:
Sure about this? Or just hidden=true? Does the helper set this forcefully or do we rely on the import magic

Reply:
```
public void mark(ImportTransactionalEntity entity) {
        entity.getImportTransactionData().setTxnId(getCurrentTransaction());
    }
    ```